environment/simulation/utils/simulation_pymunk_utils.py

```diff
@@ -22,8 +22,10 @@
 
 def create_clusters(number_of_clusters, screen_size, number_of_bots_per_threshold):
     clusters = []
-    for _ in range(number_of_clusters):
-        color = list(numpy.random.random(size=3) * 256)
+    for cluster_nr in range(number_of_clusters):
+        color = numpy.array([100, 100, 100])
```

Comment:
Dlaczego to jest array 100 na 100 na 100? Kodujemy to czymś innym niż RGB?

Reply:
To zwykły array 3x1, kwadratowy nawias to lista ktora wpada do konstruktora poprosu